### PR TITLE
Update operations.yml

### PR DESCRIPTION
**Pull request by LK112019**
Could you please add me to the OBO Foundry site?
Thanks so much!

### CODE REVIEW

**Review by LK112019 — COMMENTED** [review]
ORCID ID has been added. Thanks!

**Review by LK112019 — COMMENTED** [review]
ror# has been added

File: _data/operations.yml [hunk 1]
```diff
@@ -10,6 +10,17 @@ members:
...
   orcid: 0000-0002-1604-3078
   wikidata: Q30508557
+  - affiliation:
+    name: National Institutes of Health (NIH)
+    ror: 01cwqze88
+  country: USA
+  github: LK112019
+  groups:
+  - outreach
+  link:
```
Comment (matentzn) [inline _data/operations.yml:20]:
```suggestion
```

File: _data/operations.yml [hunk 2]
```diff
@@ -10,6 +10,17 @@ members:
...
+    name: National Institutes of Health (NIH)
+    ror: 01cwqze88
+  country: USA
+  github: LK112019
+  groups:
+  - outreach
+  link: 
+  name: Leila Kiani
+  orcid: 0009-0009-6722-6147 
+  wikidata:
```
Comment (matentzn) [inline _data/operations.yml:23]:
```suggestion
```

**Review by matentzn — CHANGES_REQUESTED** [review]
Thanks 
@LK112019 No need for the empty keys. Not sure if wikidata is required, let's see.

**Comment by cthoyt** [comment]
Photo is required. I made a wikidata for you, see #2353

File: _data/operations.yml (hunk 2 above)
Comment (cthoyt) [reply _data/operations.yml:23]:
```suggestion
  wikidata: Q118105252
```

**Review by matentzn — APPROVED** [review]

**Pull request merged**